server/test/controllers/grades.test.ts

```diff
@@ -1,11 +1,17 @@
+/* eslint-disable max-len */
```

Comment:
I think that if a linter rule is disabled, it should always have a comment explaining why it is disabled. If we disable rules all the time without reason then that kind of defeats the purpose of having a linter.

Reply:
removed lint disabling. had some test results with too long strings, fixed the length issue.